api/package-lock.json

```diff
@@ -1866,28 +1714,23 @@
     },
     "node_modules/crypto": {
       "version": "1.0.1",
-      "resolved": "https://registry.npmjs.org/crypto/-/crypto-1.0.1.tgz",
-      "integrity": "sha512-VxBKmeNcqQdiUQUW2Tzq0t377b54N2bMtXO/qiLa+6eRRmmC4qT3D4OnTGoT/U6O9aklQ/jTwbOtRMTTY8G0Ig==",
-      "deprecated": "This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in."
+      "license": "ISC"
```

Comment:
I'd consider deleting completely because it's already depreciated.

Reply:
Agree, this might be a case of node not cleaning up- that being said I didn't directly use crypto during my project